### PR TITLE
RD-5267: scrollIntoView enhancement

### PR DESCRIPTION
**Pull request by jawamartus**
## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I verified that all tests and checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

### CODE REVIEW

**Comment by jawamartus** [comment]
I had forgotten to push latest changes before. Now I did it.

File: src/components/form/Form/Form.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ interface FormProps extends Omit<StrictFormProps, 'error'> {
      * }
      * ```
      */
-    errors?: string | Record<string, string> | JSX.Element;
+    errors?: string | string[] | Record<string, string> | JSX.Element;
 
     /**
      * string with error message header
```
Comment (qooban) [inline src/components/form/Form/Form.tsx:38]: Optional:

Re. L49 - For the sake of completeness we could have single line of description for `style` prop in L49.
In other components we usually have "CSS style".

File: src/components/form/Form/Form.tsx [hunk 2]
```diff
@@ -104,8 +104,14 @@ function Form({
 }: FormProps): ReactElement | null {
     const formRef = useRef<HTMLElement>(null);
     useEffect(() => {
-        if (scrollToError && !_.isEmpty(errors)) {
-            formRef.current?.scrollIntoView?.({ behavior: 'smooth', block: 'start' });
+        if (scrollToError && formRef.current instanceof Element) {
+            const errorElement = formRef.current.querySelector('.error.field');
+
+            if (errorElement) {
+                errorElement.scrollIntoView?.({ behavior: 'smooth', block: 'start' });
```
Comment (qooban) [inline src/components/form/Form/Form.tsx:111]: As we are changing the way how scrolling to error feature works, I think we need to adjust description of `scrollToError` prop to make it clear that we either scroll to the `ErrorMessage` at the top of the `Form` or to the first erroneous `Form.Field`.

File: src/components/form/Form/Form.tsx (hunk 1 above)
Comment (jawamartus) [reply src/components/form/Form/Form.tsx:39]: Added.

File: src/components/form/Form/Form.tsx (hunk 2 above)
Comment (jawamartus) [reply src/components/form/Form/Form.tsx:124]: Modified.

**Comment by jawamartus** [comment]
Additionally I performed `npm audit fix` and updated package-lock.json .

**Review by qooban — APPROVED** [review]

**Pull request merged**